lambdas/data-migration2/src/executions.ts

```diff
@@ -91,13 +88,16 @@ export const migrateExecutions = async (
   while (record) {
     migrationSummary.dynamoRecords += 1;
 
+    if (migrationSummary.dynamoRecords % 100 === 0) {
```

Comment:
What do you think about making the logging interval come from a configurable property of `env`? And default it to 100?

`const loggingInterval = env.loggingInterval ?? 100`

Reply:
If you agree I would make the change throughout the code

Reply:
yeah, I think that's reasonable!